src/components/data/toolbar/Navigation.js

```diff
@@ -328,6 +328,7 @@ function Navigation(props) {
     const [userProfile] = useUserProfile();
     const [config] = useConfig();
     const irodsHomePath = config?.irods?.home_path;
+    const rootsQueryKeyArray = [DATA_ROOTS_QUERY_KEY, userProfile?.id];
```

Comment:
Should this also be set in a `useEffect` with `userProfile` as a dependency?

Reply:
🤔 thinking about it, yeah, I guess that's what we've done elsewhere.  It seemed to work fine locally, but I can refactor to be consistent.